### PR TITLE
chore: fix npm dependencies

### PR DESCRIPTION
**Pull request by Cifko**
Description
---
Remove unused npm dependencies. I used [depcheck](https://www.npmjs.com/package/depcheck) package

### CODE REVIEW

File: applications/launchpad/gui-vue/package.json [hunk 1]
```diff
@@ -10,25 +10,15 @@
...
     "@fortawesome/vue-fontawesome": "^3.0.0-5",
     "@oruga-ui/oruga-next": "^0.4.8",
     "@tauri-apps/api": "^1.0.0-beta.8",
     "CBuffer": "^2.2.0",
-    "core-js": "^3.6.5",
     "vue": "^3.0.0",
     "vuex": "^4.0.2"
   },
   "devDependencies": {
-    "@tauri-apps/cli": "^1.0.0-beta.10",
```
Comment (delta1) [inline applications/launchpad/gui-vue/package.json:25]: I'm really only concerned with this one - it looks like `npm run tauri` does actually use the tauri cli. Can you confirm?

**Comment by delta1** [comment]
Looks fine as long as the integration tests pass, I'm just not so sure about the changes to the launchpad

File: applications/launchpad/gui-vue/package.json (hunk 1 above)
Comment (Cifko) [reply applications/launchpad/gui-vue/package.json:25]: Yeah., I've added it back.

**Review by delta1 — APPROVED** [review]
LGTM, untested

**Comment by delta1** [comment]
bors retry

**Pull request merged**